### PR TITLE
bootstrap: add hint for gnome-shell and regen

### PR DESCRIPTION
**Pull request by jlebon**
This essentially causes `cinnamon` to drop out in favour of
`gnome-shell`. It also drops many other packages, including e.g.
`f26-backgrounds`.

### CODE REVIEW

File: data/Fedora/devel/bootstrap/aarch64/runtime-binary-packages-full.txt [hunk 1]
```diff
@@ -569,13 +560,10 @@ evolution-langpacks-3.25.91-2.fc28.noarch
 evolution-mapi-3.25.91-2.fc28.aarch64
 evolution-mapi-langpacks-3.25.91-2.fc28.noarch
 exempi-2.4.2-7.fc27.aarch64
-exiv2-libs-0.26-6.fc27.aarch64
```
Comment (yselkowitz) [inline data/Fedora/devel/bootstrap/aarch64/runtime-binary-packages-full.txt:572]: exiv2-libs will almost certainly come back later.

File: data/Fedora/devel/bootstrap/aarch64/runtime-binary-packages-full.txt [hunk 2]
```diff
@@ -911,7 +901,6 @@ google-crosextra-carlito-fonts-1.103-0.7.20130920.fc27.noarch
 google-guice-4.1-7.fc27.noarch
 google-noto-cjk-fonts-20170602-2.fc27.noarch
 google-noto-fonts-common-20161022-4.fc27.noarch
-google-noto-sans-fonts-20161022-4.fc27.noarch
```
Comment (yselkowitz) [inline data/Fedora/devel/bootstrap/aarch64/runtime-binary-packages-full.txt:914]: google-noto-sans-fonts will probably be needed at some point anyway.

File: data/Fedora/devel/bootstrap/aarch64/runtime-binary-packages-full.txt [hunk 3]
```diff
@@ -1999,6 +1988,7 @@ lvm2-dbusd-2.02.174-2.fc28.aarch64
 lvm2-devel-2.02.174-2.fc28.aarch64
 lvm2-libs-2.02.174-2.fc28.aarch64
 lvm2-lockd-2.02.174-2.fc28.aarch64
+lxpolkit-0.5.3-2.fc27.3.aarch64
```
Comment (yselkowitz) [inline data/Fedora/devel/bootstrap/aarch64/runtime-binary-packages-full.txt]: NAK!  Something else (e.g. gnome-shell or gnome-settings-daemon) should be providing PolicyKit-authentication-agent.

File: data/Fedora/devel/bootstrap/aarch64/runtime-binary-packages-full.txt [hunk 4]
```diff
@@ -3951,6 +3934,7 @@ unbound-1.6.4-3.fc27.aarch64
 unbound-libs-1.6.4-3.fc27.aarch64
 unicode-ucd-10.0.0-2.fc27.noarch
 uniconvertor-2.0-0.14.svn362.fc27.aarch64
+unique-1.1.6-18.fc27.aarch64
```
Comment (yselkowitz) [inline data/Fedora/devel/bootstrap/aarch64/runtime-binary-packages-full.txt]: This will go away once lxpolkit is dropped.

**Review by yselkowitz — CHANGES_REQUESTED** [review]

File: data/Fedora/devel/bootstrap/aarch64/runtime-binary-packages-full.txt (hunk 3 above)
Comment (tdawson) [reply data/Fedora/devel/bootstrap/aarch64/runtime-binary-packages-full.txt]: I think it is polkit-gnome

Comment (yselkowitz) [reply data/Fedora/devel/bootstrap/aarch64/runtime-binary-packages-full.txt]: Nowadays it's provided by gnome-shell:

https://git.gnome.org/browse/gnome-shell/tree/src/shell-polkit-authentication-agent.c

File: archful-srpms.txt [hunk 5]
```diff
@@ -4,8 +4,6 @@ bcc
 binutils
 boost
 ceph
-cinnamon
```
Comment (igor-raits) [inline archful-srpms.txt]: you can keep those, they don't make any harm

**Pull request merged**